### PR TITLE
Add D202 pydocstyle check

### PR DESCRIPTION
**Pull request by pcandoalmeida**
Enabled D202 Pydocstyle check

* Fixed all pre-commit errors

Part of issue: #10742

---

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/providers/amazon/aws/hooks/glue_catalog.py [hunk 1]
```diff
@@ -104,12 +104,10 @@ def get_table(self, database_name, table_name):
         :param table_name: Name of hive table
         :type table_name: str
         :rtype: dict
-
         >>> hook = AwsGlueCatalogHook()
```
Comment (mik-laj) [inline airflow/providers/amazon/aws/hooks/glue_catalog.py]:
```
        """
        Get the information of the table

		.. code-block:: py

            hook = AwsGlueCatalogHook()
            r = hook.get_table('db', 'table_foo')
            r['Name'] = 'table_foo'

        :param database_name: Name of hive database (schema) @table belongs to
        :type database_name: str
        :param table_name: Name of hive table
        :type table_name: str
        :rtype: dict
        """
```
Can you try it?

Comment (pcandoalmeida) [reply airflow/providers/amazon/aws/hooks/glue_catalog.py]: Yep, that works, thanks @mik-laj!

**Review by kaxil — APPROVED** [review]
LGTM. Good work @pcandoalmeida

**Comment by kaxil** [comment]
Let me know when you are happy to mark the "PR ready for review" -- I think it is but just want to make sure :)

**Comment by pcandoalmeida** [comment]
Hi @kaxil sorry, I was trying to find out why I was getting a K8s error. I'll see if I get an error with the latest CI run, might need to rebase.

**Comment by kaxil** [comment]
Yeah don't worry about the Kubernetes tests,  they are unrelated to the PR, looks like some transient error which we will look into.

**Comment by kaxil** [comment]
Can you rebase to the master, it should be better now

**Comment by pcandoalmeida** [comment]
OK sure @kaxil 😃

**Pull request merged**